src/main/java/org/jenkinsci/plugins/durabletask/executors/ContinuedTask.java

```diff
@@ -60,6 +76,10 @@ private static boolean isContinued(Queue.Task task) {
             if (isContinued(item.task)) {
                 return null;
             }
+            Jenkins jenkins = Jenkins.getInstance();
+            if (jenkins == null || jenkins.getInitLevel().compareTo(InitMilestone.COMPLETED) < 0) {
```

Comment:
@stephenc would be cool if you could add more comments into Queue code/methods/classes as it  seems only authors knows what is happening in queue.